lib/server.js

```diff
@@ -51,6 +51,15 @@ function Server (torrent, opts) {
     // deny them
     if (req.headers.origin == null) return false
 
+    // If a 'hostname' string is specified, deny requests with a 'Host'
+    // header that does not match the origin of the torrent server to prevent
+    // DNS rebinding attacks.
+    if (opts.hostname) {
+      if (req.headers.host !== `${opts.hostname}:${server.address().port}`) {
```

Comment:
What would Brave set the `hostname` option to? If it would be set to `chrome-extension://xxx`, then I fear this won't work since it's going to append the torrent server's port to that and reject anything that doesn't match that.

I think we can remove the code that adds the port. Correct me if I'm wrong.

Reply:
@feross actually it would set it to `localhost`. The expected host is `localhost:$RANDOM_PORT`, as assigned in https://github.com/dcposch/webtorrent-remote/blob/3e99afb9a58787afb68f780da0a8f3174aa6cf0e/server.js#L203

note that the Host header is the origin of the server that the client is trying to contact